demo-app/src/pages/locations-map.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Anchor, Page, PageContent, PageHeader, Footer } from 'grommet';
+import POCMap from '../POCMap';
+
+const BasicMap = () => {
```

Comment:
What is the purpose of this page vs the one on home page? Just prepping for the GLCP use case?

Reply:
Yes, want to be able to set up the demo app to navigate to different types/varieties of map experiences, rather than be locked into a single experience.

Home will become a hub.

Reply:
I'm fine leaving for now